src/pages/items/ItemsPage.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from "react";
+import { getItems } from "../../api/ItemApi";
+import PandaBestList from "./PandaBestList";
+import PandaAllList from "./PandaAllList";
+
+function ItemsPage() {
+  const [items, setItems] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const data = await getItems();
+        setItems(data);
+        setLoading(false);
+      } catch (err) {
+        setError(err.message);
+        setLoading(false);
+      }
```

Comment:
로딩같은 경우 에러든 아니든 결국 로딩 완료되기에 finally 로 받아주면 좋을 거 같아요!

```suggestion
      try {
        const data = await getItems();
        setItems(data);
      } catch (err) {
        setError(err.message);
      } finally {
        setLoading(false);
      }
```